charts/redpanda/temp-template.yaml

```diff
@@ -0,0 +1,1528 @@
+---
```

Comment:
Should this file be committed?

Reply:
Looks like an accidental commit. Please remove.